templates/envproxy.yaml

```diff
@@ -70,7 +70,7 @@ spec:
 {{- end}}
           securityContext:
             allowPrivilegeEscalation: false
-            readOnlyRootFilesystem: true
+            readOnlyRootFilesystem: {{ .Values.envproxy.securityContext.readOnlyRootFilesystem }}
```

Comment:
Can we do: `{{ .Values.envproxy.securityContext.readOnlyRootFilesystem | default true }}` here? This way users don't need to add it to their values file if they don't already have it in there https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function

Reply:
Is that not achieved by giving it a default value of `true` in the values.yaml?

Reply:
I'm not sure actually, does it always apply the values.yaml included in the chart first, and then the user's values?

Reply:
Yup, if you specify nothing in your "override" it'll take the values in the packaged `values.yaml`

Reply:
cool stuff, thanks for the info